public/pages/CreateMonitor/containers/AnomalyDetectors/AnomalyDetectors.js

```diff
@@ -0,0 +1,110 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { FormikComboBox } from '../../../../components/FormControls';
+import { hasError, isInvalid, validateDetector } from '../../../../utils/validate';
+import { AppContext } from '../../../../utils/AppContext';
+
+class AnomalyDetectors extends React.Component {
+  static contextType = AppContext;
+  constructor(props) {
+    super(props);
+    this.state = {
+      detectorOptions: [],
+      isLoading: false,
+    };
+    this.searchDetectors = this.searchDetectors.bind(this);
+  }
+  async componentDidMount() {
+    await this.searchDetectors();
+  }
+
+  async searchDetectors() {
+    const { httpClient } = this.context;
+    try {
+      const response = await httpClient.post('../api/alerting/detectors/_search');
+      if (response.data.ok) {
+        const detectorOptions = response.data.detectors.map(detector => ({
+          label: detector.name,
+          value: detector.id,
+          features: detector.featureAttributes,
+          //TODO::Temp hack until backend fix
```

Comment:
make github issue

Reply:
This is not a problem now. Will remove this TODO